<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Eisenbahn Fernsteuerung</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="selectEngine">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="engine1">
         <attribute name="title">
          <string>Intercity</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="speed1plus">
              <property name="text">
               <string>Geschwindigkeit erhöhen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speed1minus">
              <property name="text">
               <string>Geschwindigkeit verringern</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop1">
              <property name="text">
               <string>Stopp!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="speed1"/>
            </item>
            <item>
             <widget class="QSlider" name="slider1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="engine2">
         <attribute name="title">
          <string>Stainz I</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="speed2plus">
              <property name="text">
               <string>Geschwindigkeit erhöhen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speed2minus">
              <property name="text">
               <string>Geschwindigkeit verringern</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop2">
              <property name="text">
               <string>Stopp!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="speed2"/>
            </item>
            <item>
             <widget class="QSlider" name="slider2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="engine3">
         <attribute name="title">
          <string>Stainz II</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="speed3plus">
              <property name="text">
               <string>Geschwindigkeit erhöhen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speed3minus">
              <property name="text">
               <string>Geschwindigkeit verringern</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop3">
              <property name="text">
               <string>Stopp!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="speed3"/>
            </item>
            <item>
             <widget class="QSlider" name="slider3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="engine4">
         <attribute name="title">
          <string>E-Lok</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="speed4plus">
              <property name="text">
               <string>Geschwindigkeit erhöhen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speed4minus">
              <property name="text">
               <string>Geschwindigkeit verringern</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop4">
              <property name="text">
               <string>Stopp!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="speed4"/>
            </item>
            <item>
             <widget class="QSlider" name="slider4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="engine5">
         <attribute name="title">
          <string>Bachmann</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QPushButton" name="speed5plus">
              <property name="text">
               <string>Geschwindigkeit erhöhen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="speed5minus">
              <property name="text">
               <string>Geschwindigkeit verringern</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop5">
              <property name="text">
               <string>Stopp!</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="speed5"/>
            </item>
            <item>
             <widget class="QSlider" name="slider5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QPushButton" name="emergencystop">
      <property name="text">
       <string>Nothalt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>532</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMen">
    <property name="title">
     <string>Menü</string>
    </property>
    <addaction name="actionVerbindung_testen"/>
    <addaction name="actionBeenden"/>
   </widget>
   <addaction name="menuMen"/>
  </widget>
  <action name="actionVerbindung_testen">
   <property name="text">
    <string>Verbindung testen</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
